aspnetcore/fundamentals/index.md

```diff
@@ -125,7 +125,13 @@ For more information, see <xref:fundamentals/host/index>.
 
 ## Servers
 
-The ASP.NET Core hosting model doesn't directly listen for requests. The hosting model relies on an HTTP server implementation to forward the request to the app. The forwarded request is wrapped as a set of feature objects that can be accessed through interfaces. ASP.NET Core includes a managed, cross-platform web server, called [Kestrel](xref:fundamentals/servers/kestrel). Kestrel is commonly run behind a production web server, such as [IIS](https://www.iis.net/) or [Nginx](http://nginx.org) in a reverse proxy configuration. Kestrel can also be run as a public-facing edge server exposed directly to the Internet in ASP.NET Core 2.0 or later.
+The ASP.NET Core hosting model doesn't directly listen for requests. The hosting model relies on an HTTP server implementation to forward the request to the app.
+
+ASP.NET Core provides the following server implementations:
+
+* [Kestrel](xref:fundamentals/servers/kestrel) is a managed, cross-platform web server. Kestrel is often run behind a production web server in a reverse proxy configuration, such as [IIS](https://www.iis.net/), [Nginx](http://nginx.org), or [Apache](https://httpd.apache.org/). Kestrel can also be run as a public-facing edge server exposed directly to the Internet in ASP.NET Core 2.0 or later.
```

Comment:
Introduce the 3 OS tabs here. The macOS and Linux tabs would only contain Kestrel. The Windows tab would contain all 3 options. That makes things easier to scan for macOS and Linux customers. The best part is that the reader's OS is the default selected tab.